region/tourian/main/Metroid Room 1.json

```diff
@@ -858,15 +996,95 @@
             "Gravity",
             {"acidFrames": 10}
           ]}
+        ]},
+        {"or": [
+          {"obstaclesCleared": ["A"]},
+          "Ice",
+          "ScrewAttack",
+          "canMetroidAvoid",
+          {"and": [
+            "canPseudoScrew",
+            "canTrickyJump"
+          ]},
+          {"metroidFrames": 420}
+        ]},
+        {"or": [
```

Comment:
I wasn't sure what this `or` was for. Is HiJump another option?

Reply:
It is not, but the jump is a little easier than I thought so I added another option